msal/src/main/java/com/microsoft/identity/nativeauth/NativeAuthPublicClientApplication.kt

```diff
@@ -582,7 +582,7 @@ class NativeAuthPublicClientApplication(
                     return@withContext SignUpError(
                         errorType = ErrorTypes.INVALID_USERNAME,
                         errorMessage = "Empty or blank username",
-                        correlationId = "UNSET"
+                        correlationId = DiagnosticContext.INSTANCE.threadCorrelationId
```

Comment:
CommandDispatcher.submitSilentReturningFuture will clear the context at end every time.